### PR TITLE
Atlas - adding configuration files + mfapp addon

### PR DESCRIPTION
**Pull request by pmauduit**
Atlas is not released yet but having the file already present here will make the development easier.

### CODE REVIEW

**Comment by fvanderbiest** [comment]
This PR should target master.
Master is being merged into docker-master ...

**Comment by stlejer-morvrini** [comment]
On 21/06/16 16:04, François Van Der Biest wrote:
> This PR should target master.
> Master is being merged into docker-master ...

Atlas: field papers inside? ;D
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/georchestra/datadir/pull/17#issuecomment-227449712>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACj973lPXFweSP3QyCblSZ5cMQV6qe83ks5qN-9QgaJpZM4I6vSa>.
> 


-- 
Rod Béra,  MCF Géomatique                /   Lecturer, Geomatics
           et SIG pour l'Environnement  /    and Environmental GIS
Agrocampus-Ouest|65 r.Saint-Brieuc|CS84215|35042 Rennes cedex|France
+33 (0) 223 48 5553 - roderic.bera@agrocampus-ouest.fr

File: atlas/logback/logback.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<configuration>
+
+    <appender name="standardOut" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <Pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</Pattern>
+        </encoder>
+    </appender>
+
+    <logger name="org.mapfish" level="WARN" />
+    <logger name="ch.thus.camel.retry" level="WARN" />
```
Comment (Vampouille) [inline atlas/logback/logback.xml]: I think this one is not used

Comment (pmauduit) [inline atlas/logback/logback.xml]: It actually is:
https://github.com/georchestra/georchestra/blob/Atlas/atlas/src/docker/Dockerfile#L8

Comment (Vampouille) [inline atlas/logback/logback.xml]: This entry : `ch.thus.camel.retry` come from mapfish batch, which is not include in atlas

Comment (pmauduit) [inline atlas/logback/logback.xml]: Sorry I misunderstood :)

**Comment by landryb** [comment]
I see that Atlas uses logback.. maybe that would make sense to reconsider switching to it globally to enabled MDC (georchestra/georchestra#1149) :)

**Comment by fvanderbiest** [comment]
Closing since atlas datadir files have been pushed to master with https://github.com/georchestra/datadir/commit/f47517d4e21328b5c8a0287e67172d6009c9b445

**Comment by fvanderbiest** [comment]
Changes merged into `docker-master` with 8a10693 
Thanks @pmauduit

**Pull request closed without merging**